tests/test_type_inference/test_initializer.py

```diff
@@ -0,0 +1,62 @@
+import astroid
+from typing import _ForwardRef
+from python_ta.transforms.type_inference_visitor import TypeInferer
+from python_ta.typecheck.base import TypeInfo, TypeFail
+from nose.tools import eq_
+
+
+def test_class_with_init():
+    program = """
+    class Foo:
+        a
+        
+        def __init__(self):
+            a = 5
+    
+    foo = Foo()
+    """
+    ti = TypeInferer()
```

Comment:
These tests should all use the standard test helper functions. See other test files for examples.